cli/azd/resources/starter/infra/keyvault.bicep

```diff
@@ -0,0 +1,17 @@
+param name string
```

Comment:
I think you are trying to include a sample with main.bicep and keyvault.bicep that works. My worry here is customer may get confused.